contracts/staking/BaseShareField.sol

```diff
@@ -82,9 +83,10 @@ contract BaseShareField is DAOContract{
     
     
     /**
-    * @dev Audit user's rewards and calculate their earned reward based on
-    * If user's stake time more than one month so it calculated with 0.5x 
-    * multiplier otherwise for over 1 month part it gets 1x multiplier
+    * @dev Audit user's rewards and calculate their earned rewards based on
+    * If user's stake time less than one month so it calculated with 0.5x 
```

Comment:
its not if stake time is less, for first month it is always 0.5x

Reply:
Replaced